lua/nvim-treesitter/parsers.lua

```diff
@@ -1367,6 +1367,17 @@ list.scala = {
   maintainers = { "@stevanmilic" },
 }
 
+list.scfg = {
+  install_info = {
+    url = "https://git.sr.ht/~rockorager/tree-sitter-scfg",
+    files = { "src/parser.c" },
+    requires_generate_from_grammar = true,
+    branch = "main",
```

Comment:
That is not necessary, if it's the default branch. You do, however, have to add a lockfile entry.

Reply:
> That is not necessary, if it's the default branch.

`TSInstall scfg` fails if I don't specify this:

```
nvim-treesitter[scfg]: Error while checking out revision                                      
error: pathspec 'master' did not match any file(s) known to git
```

I think it doesn't use the default branch, but `master` instead.

Reply:
Yes, as I said, you need to add a lockfile entry.